test/urlRouterSpec.js

```diff
@@ -84,6 +108,12 @@ describe("UrlRouter", function () {
       expect(match[0]).toBe("/foo/:param");
       expect(match[1]).toEqual({param: 'baz'});
     });
+    
+    it('should default to the first rule defined for same url', function() {
```

Comment:
This test always passed in PhantomJs, but when run in Chrome (`grunt karma:debug`) this test would fail (before I implemented the stable sort fix).